opencti-platform/opencti-graphql/src/graphql/loggerPlugin.js

```diff
@@ -29,7 +29,7 @@ const tryResolveKeyPromises = async (data) => {
   }
 };
 
-const API_CALL_MESSAGE = 'API Call'; // If you touch this, you need to change the performance agent
+const API_CALL_MESSAGE = 'GRAPHQL_API'; // If you touch this, you need to change the performance agent
```

Comment:
Did you changed the performance agent ? I have no idea on what is the performance agent on my side.

Reply:
Yes, it will be impacted after the merge.
Performance agent is the automated internal performance tests of the solution.